lib/rucio/core/rule.py

```diff
@@ -750,10 +754,11 @@ def list_rules(filters={}, *, session: "Session"):
             query = query.filter(getattr(models.ReplicationRule, key) == value)
 
     try:
-        for rule in query.yield_per(5):
+        for rule, data_identifier_bytes in query.yield_per(5):
             d = {}
             for column in rule.__table__.columns:
                 d[column.name] = getattr(rule, column.name)
+            d['size'] = data_identifier_bytes
```

Comment:
I wonder whether `bytes` is better than `size` because it’s more consistent with all other attributes that represent volume.